pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/widgets/tkg-labels/interfaces/tkg-labels.interface.ts

```diff
@@ -0,0 +1,21 @@
+import {FormArray, FormGroup} from '@angular/forms';
+
+export interface TKGLabelsConfig {
+    label: {
+        title: string;
+        tooltipText: string;
+        helperText?: string;
+    };
+    forms: {
+        parent: FormGroup; // the parent form group
+        control: FormArray; // the control of the labels form array
+    };
+    fields: {
+        [key: string]: any;
+    };
+}
+
+export interface TKGLabel<K = string, V = string> {
```

Comment:
Is there a case where we would use TKGLabel for a type other than string?

Reply:
refactored to use string